content/en/vision.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Aurora Architecture and Vision"
+date: 2025-02-05
+draft: false
+sidebar: false
+showToc: true
+translationKey: "vision"
+---
+
+## Overview
+
+One of the goals of the Aurora team is to design and build a cloud agnostic platform that supports teams looking to take advantage of "cloud-first" designs.
+
+To achieve this, the Aurora team provides a service offering known as the Aurora Application Hosting Platform (AUR).
+
+To highlight the benefits, goals and purpose of Aurora the Aurora team has created a product vision board.
+
+![Aurora Product Vision](/images/aurora/product-vision.png "Aurora Product Vision")
+
+**Figure 1** - The Aurora Product Vision Board
+
+The product vision board is used to provide a common understanding of the direction to move towards and it guides and informs decisions regarding Aurora.
+
+> The production vision board guides the Aurora team in the overal direction of where we are going, and at times might not necessarily reflect what is available today
+
+## Anatomy of the Product Vision Board
+
+The product vision board consists of five sections.
+
+1. **Vision:** the goal we want to achieve
+1. **Target Group:** the group most likely to benefit from the using the product
+1. **Needs:** the provided benefit of using the product
+1. **Product:** key features of the product
+1. **Business Goals:** benefits to the organisation
+
+### Vision
+
+Aurora unifies the best of both worlds open-source and proprietary Kubernetes environments, offering a standardized, scalable, and secure hosting solution.
+
+### Goals
+
+By 2028, Aurora will be the cornerstone of application hosting for Canada’s federal government, uniting departments under a secure, cloud-agnostic platform designed for innovation and operational excellence. It will streamline the onboarding process with intuitive self-service capabilities, ensure consistent governance across diverse environments, and empower a community of certified architects and solution builders to drive digital transformation. By prioritizing scalability, security, and interoperability, Aurora will redefine how government solutions are developed, deployed, and managed, fostering collaboration and setting a new benchmark for modern public-sector technology platforms.
+
+### Target Group
+
+"Solution builders" is the umbrella term that encompasses both traditional Information Technology (IT) users, such as application developers as well as non-traditional IT users, such as data scientists and data analysts. The term "solution" is used instead "app" or "system" to better reflect the range of solutions that could be available on the CNP. For example, this can include large-scale solutions for corporate initiatives as well as smaller solutions that may be standalone and serve very specific purposes.
```

Comment:
```suggestion
"Solution builders" is the umbrella term that encompasses both traditional Information Technology (IT) users, such as application developers as well as non-traditional IT users, such as data scientists and data analysts. The term "solution" is used instead "app" or "system" to better reflect the range of solutions that could be available on Aurora. For example, this can include large-scale solutions for corporate initiatives as well as smaller solutions that may be standalone and serve very specific purposes.
```